modules/apps/site-initializer/site-initializer-liferay-marketplace/site-initializer-liferay-marketplace-test/src/testFunctional/paths/MP2Utils.path

```diff
@@ -0,0 +1,27 @@
+<html>
+<head>
+<title>MP2Utils</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">MP2Utils</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>NEW_MEMBER_BUTTON</td>
+	<td>//button[contains(text(),'+ New Member')]</td>
```

Comment:
Hey @LuanaKSilva, the path `NEW_MEMBER_BUTTON` can be more general with a variable, because it works in other sections, e.g. in Projects in the '+ New Project' button and in Apps in the '+ New App' button.

Reply:
Oh yes, as only the dashboard flow works today, I made it just thinking about it. But it is an excellent idea, I will make it more generic. Thanks!